docs/source/variables.ipynb

```diff
@@ -0,0 +1,182 @@
+    "\n",
+    "- Gender\n",
+    "- Eye Color\n",
+    "- Nationality\n",
+    "- Hair Color\n",
+    "- Spoken Language\n",
+    "\n",
+    "### Ordinal\n",
+    "\n",
+    "A Ordinal variable type can take ordered discrete values. Similar to Categorical, it is usually a limited, and fixed number of possible values. However, these discrete values have a certain order, and the ordering is important to understanding the values. Ordinal variable types can be represented as strings, or integers. \n",
+    "\n",
+    "Some examples of Ordinal variable types:\n",
+    "\n",
+    "- Educational Background (Elementary, High School, Undergraduate, Graduate)\n",
+    "\n",
+    "- Satisfaction Rating (“Not Satisfied”, “Satisfied\", “Very Satisfied”)\n",
+    "\n",
+    "- Spicy Level (Hot, Hotter, Hottest)\n",
+    "\n",
+    "- Student Grade (A, B, C, D, F)\n",
+    "\n",
+    "- Size (small, medium, large)\n",
+    "\n",
+    "\n",
+    "#### Categorical SubTypes (CountryCode, Id, SubRegionCode, ZIPCode)\n",
+    "\n",
+    "There are also more distinctions within the Categorical variable type. These include CountryCode, Id, SubRegionCode, and ZIPCode.\n",
+    "\n",
+    "It is important to make this distinction because there are certain operations that can be applied, but they don't necessary apply to all Categorical types. For example, there could be a [custom primitive](https://docs.featuretools.com/en/stable/automated_feature_engineering/primitives.html#defining-custom-primitives) that applies to the ZIPCode variable type. It could extract the first 5 digits of a ZIPCode. However, this operation is not valid for all Categorical variable types. Therefore it is approriate to use the ZIPCode variable type. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Datetime\n",
+    "A Datetime is a representation of a date and/or time. Datetime variable types can be represented as strings, or integers. However, they should be in a intrepretable format or properly cast before using DFS. \n",
+    "\n",
+    "Some examples of Datetime include:\n",
+    "\n",
+    "- transaction time\n",
+    "- flight departure time\n",
+    "- pickup time\n",
+    "\n",
+    "### DateOfBirth\n",
+    "A more distinct type of datetime is a DateOfBirth. This is an important distinction because it allows additional primitives to be applied to the data to generate new features. For example, having an DateOfBirth variable type, will allow the Age primitive to be applied during DFS, and lead to a new Numeric feature."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## TextT\n",
+    "Text is a long-form strings, that can be of any length. It is commonly used with NLP operations, such as TF-IDF. Featuretools supports NLP operations with nlp-primitives [add-on](https://innovation.alteryx.com/natural-language-processing-featuretools/). Text variable type can be represented as strings. "
```

Comment:
I think the last sentence is repeating information found in the first sentence.

1st and 3rd sentence could use an extra word or two.